api/envoy/api/v2/route/route.proto

```diff
@@ -773,6 +773,22 @@ message RouteAction {
     google.protobuf.BoolValue enabled = 2;
   };
   repeated UpgradeConfig upgrade_configs = 25;
+
+  // Configures :ref:`internal redirect <arch_overview_internal_redirects>` behavior.
+  enum RedirectAction {
+    PASS_THROUGH_REDIRECT = 0;
```

Comment:
Do you mind adding documentation for all the enum values so it's easier to review the options inline?